ecs-init/docker/docker_test.go

```diff
@@ -253,6 +253,14 @@ func validateCommonCreateContainerOptions(opts godocker.CreateContainerOptions,
 	if hostCfg.NetworkMode != networkMode {
 		t.Errorf("Expected network mode to be %s, got %s", networkMode, hostCfg.NetworkMode)
 	}
+
+	if len(hostCfg.CapAdd) != 1 {
```

Comment:
We can refactor the tests and make them use the assert package. 
We can track this as a new [issue](https://github.com/aws/amazon-ecs-init/issues/98)